css/style.css

```diff
@@ -1,7 +1,355 @@
+/* ----------- VISUALIZATION ----------- */
+
 .circle {
   display: inline-block;
   width: 100px;
   height: 100px;
   border-radius: 50px;
   background-color: red;
 }
+
+/* ----------- UI - General ----------- */
+
+*::before, *::after, * {
+  box-sizing: border-box;
+}
+
+body {
+  display: grid;
+  grid-template-columns: 170px auto;
+  height: 100vh;
+  margin: 0;
+  font-family: "Open Sans", sans-serif;
+  font-weight: normal; 
+}
+
+h1, h2, h3 {
+  margin: 0;
+  font-weight: 400;
+}
+
+h1 {
+  font-size: 1.75em;
+}
+
+h2 {
+  font-size: .75em;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+h3 {
+  font-size: 1.2em;
+  color: #73A4B8;
+  text-transform: uppercase;
+}
+
+button {
+  position: relative;
```

Comment:
Using relative and absolute here? Is this for animations? Absolute really should be avoided at most costs.